exonum-light-client/pom.xml

```diff
@@ -61,7 +61,7 @@
 
   <properties>
     <!-- Project configuration -->
-    <ejb.version>0.6.0</ejb.version>
+    <ejb.version>0.7.0-SNAPSHOT</ejb.version>
```

Comment:
Why if there is no such need (= there are no things from 0.7.0-s the light client needs)? It will make it impossible to, say, release 0.3.0 if 0.7.0 has not come out yet (= 0.3.0 will require downgrading to 0.6.0, switching the version back and forth).